### PR TITLE
Added negroski 

### PR DESCRIPTION
**Pull request by Guberlo**
related issue #882

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, is it possible to use another image that differs from the `negroni` recipe we have already?

**Comment by Guberlo** [comment]
Hi @alfg, the picture I added Is different from the other two. 
Maybe you are confusing It with ```negroni-sbagliato.jpg```?

File: src/recipes/negroski.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "ingredient": "Orange peel"
+        }
+    ],
+    "directions": [
+        "Chill a rock medium or old fashioned glass.",
+        "Fill the glass with large ice cubes.",
+        "Pour the spirits and give a good stir.",
+        "Garnish with an orange peel."
+    ],
+    "image": "negroni.jpg",
```
Comment (alfg) [inline src/recipes/negroski.json:33]:
```suggestion
    "image": "negroski.jpg",
```

**Comment by alfg** [comment]
@Guberlo Ah, I see it now. It seems the JSON file's image property is referring to the `negroni.jpg` image.

**Comment by Guberlo** [comment]
Ups, my fault!
I've commited your suggestion now 👍

**Comment by alfg** [comment]
Thanks!

**Pull request merged**